### PR TITLE
Created problem_55 in project_euler

### PR DESCRIPTION
**Pull request by Kush1101**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: project_euler/problem_55/sol1.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+That is, 349 took three iterations to arrive at a palindrome.
+Although no one has proved it yet, it is thought that some numbers, like 196,
+never produce a palindrome. A number that never forms a palindrome through the
+reverse and add process is called a Lychrel number. Due to the theoretical nature
+of these numbers, and for the purpose of this problem, we shall assume that a number
+is Lychrel until proven otherwise. In addition you are given that for every number
+below ten-thousand, it will either (i) become a palindrome in less than fifty
+iterations, or, (ii) no one, with all the computing power that exists, has managed
+so far to map it to a palindrome. In fact, 10677 is the first number to be shown
+to require over fifty iterations before producing a palindrome:
```
Comment (cclauss) [inline project_euler/problem_55/sol1.py]:
```suggestion
to require over fifty iterations before producing a palindrome:

```

**Comment by Kush1101** [comment]
@cclauss Please review it.

**Review by Kush1101 — COMMENTED** [review]
@cclauss. Is there a permanent solution  to resolve the small mistakes like trailing whitespaces in comments so the tests don't fail. The [Black editor](https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4Am0BDZdAD2IimZxl1N_Wk8JgdwWhs5zsNRzNH6DXnxoRWop43JRhKWyBx405fFky-fJ9GQkDZV-j4sgc6EwIJyo8rLB6FvTFcThqwhKf96-MmIjFOpan3nf7LTNsnTWD2DeXkdyl-8ej5mlHJZd88mTABhljXRbM8OgpQE3Sbg-x5yozIfTITPfubQN4eFDssMON9Jz29l4vyOQV8N2hTEMVdf9l8AgjPjPA2FMGkgun5yovYMUf8hYdGTFyc465xPcEQYo1hKAne7pZ1cdrx2uopN5cdKvsrT1l90qoxWIbeqZX-M_pUECtdj7U6lQPgAZJ2f1SPN_mt7kYPxSbR-Jsy8K__9DczRdzvZGOnqEkjCqu8y2Gs6SXWC5J83eseIT2Hw_gv2D-VqHJ12nhzafgUcLgnbLON9yWccCkUMW-9kJVYguD302hmTEbbyFuNaTK9CIPgKXk6NQdvSS1vq4jWtaNjYaW74BCD0JGrjHs9OoPY252pOluZy92ybDkc6sZWTfGHTC4SZcWxrs0D7NywKkTz-XGUyyl-xJDMyl74bFL9QbbzVBiKyUfdZm_hncaGnJEmdlFVh5YFn_6Jbu0_5xWhK-PrL0etOK-_P5OTnYs9ucHRp6K1R0Ia8jaAktg6eDzftwo0iUq2pTjYMYskVoGPSP8Wm7P8cogD3jutr7vVMvcPrQKu-JRCp8nKFXAdcYe985Qwm81Vqmnq5wyDV15oA9uY3wOHXptrnkeZQxIucJzlxd0BAhi_1I1TTFDLRI1OcMZtaG-jKKXvko3vFuXgH5zFW4un6MTv_3XcPhVcamfxWCbeCVWamZ_gPkRUfA6NtVsWLbo-Gpr-cnfPl7vLhPtQ8IFe898jliEjEGxMoKe1wnXYiN5QnE33QsQ7Tx7P3aRvClqntoy_sGYJ02IQlW5qzaywLZ-f4CZyA7O3YeW5OldoMxt_nDNBOuksUONQQQITLrkZVOXeD-9FFSavHHGRtue_NrxkzfqLFxqb9am5dlCLhAsDPB0yFQjvqNAh1i8I4WcXykDSiRMCyuxmMsPngq0lvyoGPcul4DlkDY0BpqXjaMYph7TcsveMF3Z4HpWqGoJhO7SkrSQiYPrz4NJiPitAmhxAc8d3m430_gUd5Z20tJ-l-EdHSsPth0jXUJSF5DLcVbBOzFC_mD2dSSCjd-1CqmFeybn9IBh8hhLvFcwqBDipUx0c_7FdLB67NqR4iW44W-dUItFu6t-vYmf6kHPJ2GAYO8dj1RCmiKakBMKsN4K31cwS997SDZUTLnhGEYvLtg6txn2WbwH_rdstpYsCEDArhxZS6tu19vMqR4Y9cIhffiRdmLfzB8x412RqV0x4wMZbK-FRfwFAsCV2L6flAEyD14a2tUEEhEuMmiuPpCIR3dyfioFL7IkNFk_2-UrC5y8ATcQ9jPSwGOAwtYiAAAAACP7cPXFfHhQQAB0gi1EwAAuYEfmLHEZ_sCAAAAAARZWg==) cannot help with this.

**Comment by cclauss** [comment]
There is a setting in VS Code (and other editors) that will remove trailing whitespace.  I think black will not modify text that is inside a triple quoted string.

**Review by cclauss — APPROVED** [review]

**Pull request merged**